src/Controller/Admin/MemberCrud.php

```diff
@@ -50,15 +50,14 @@ public function configureCrud(Crud $crud): Crud
         return $crud
             ->setEntityLabelInSingular('lid')
             ->setEntityLabelInPlural('Leden')
-            ->setSearchFields(['id', 'firstName', 'lastName', 'email', 'phone', 'city', 'postCode', 'currentMembershipStatus.name'])
+            ->setSearchFields(['id', 'firstName', 'lastName', 'email', 'phone', 'city', 'postCode'])
         ;
     }
 
     public function configureFilters(Filters $filters): Filters
     {
-        return $filters
-            ->add(EntityFilter::new('division'))
-            ->add(EntityFilter::new('currentMembershipStatus'));
+        $filters->add(EntityFilter::new('division'));
+        return $filters;
```

Comment:
Is er een reden dat je deze weghaalt? In #162 breid ik deze juist uit.

Reply:
Nee, dit is fout gegaan bij het rebasen. Ik heb een nieuwe commit gepushed om dit te corrigeren